core-9/src/main/java/SpringApp.java

```diff
@@ -8,11 +8,30 @@
 // 4. Create FordConveyor interface with a method Car produceFordCar();
 // 5. Implement two different conveyors with creating FordBronco prototype beans.
 
+import configuration.BeanConfiguration;
+import configuration.car.FordConveyorA;
+import configuration.car.FordConveyorB;
+import configuration.car.carImpl.FordBronco;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 
 public class SpringApp {
 
     public static void main(String[] args) {
         //initialize application context there
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext();
+        context.register(BeanConfiguration.class);
+        context.getEnvironment().setActiveProfiles("old", "sport", "cross-country");
+        context.refresh();
+        context.registerShutdownHook();
+
+        FordConveyorA fordConveyorA = context.getBean("fordConveyorA", FordConveyorA.class);
+        FordConveyorB fordConveyorB = context.getBean("fordConveyorB", FordConveyorB.class);
+
+        FordBronco fordBroncoA = (FordBronco) fordConveyorA.produceFordCar();
+        FordBronco fordBroncoB = (FordBronco) fordConveyorB.produceFordCar();
+
+        System.out.println(fordBroncoA == fordBroncoB);
```

Comment:
not really a good check, try it this way - (FordBronco) fordConveyorA.produceFordCar() == (FordBronco) fordConveyorA.produceFordCar(), it should be false everytime

the goal of a real conveyor is to assemble a new car every time, not the same